mollie-payments-for-woocommerce/includes/mollie/wc/gateway/mistercash.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Class Mollie_WC_Gateway_MisterCash
+ *
+ * LEGACY - DO NOT REMOVE!
+ * MiserCash was renamed to Bancontact, but this class should stay available for
```

Comment:
MiserCash -> MisterCash.

Can you also add a `@deprecated Replaced by Mollie_WC_Gateway_Bancontact`? Using this class in a IDE will show the class as deprecated (with a deprecation message).

Reply:
https://github.com/mollie/WooCommerce/commit/ab5bd8873a30cc514ab291df51b8531dc0be8415